### ISSUE TITLE
Formatierung von Zotero-Zitaten

### ISSUE THREAD

**Issue by sviatoplok** [sviatoplok/mrp-regeln#21]
Wenn ich eine `{Citation}` über Zotero angelegt habe:

ist es sinnvoll, die mit MRP.dot zu formatieren, oder machst du das in einem weiteren Schritt?

Grund: die Formatierung geht früher oder später wieder verloren und stört evtl. die Funtion von Zotero

**Comment by skurzinz** [comment]
Nein, ist nicht sinnvoll. Das Formatting von Zotero-Einträgen müssen wir über das CSL lösen (cf. #18). 

Welche Dinge aber NICHT als direkte Zotero-Einträge reinkommen, kann ich hier nicht genau feststellen, ausgeschlossen können jedenfalls werden: 

- Zeitungen (Anno-Links pro Datum) -- hier muss am Schluss ein Rahmeneintrag "Neue Freie Presse" usw. angelegt werden mit dem Zweck der Print-Ausgabe der Bibliografie (und der `tei:listBibl`). Hier legen wir nicht einzelne Artikel usw. an.  Hier kein Formatting erforderlich, auch nicht via Word-Formatvorlagen ("MRP.dotx")
  - Bei Akten- und Gesetzessammlungen müssen wir analog "Rahmeneinträge" anlegen, aber einzelne Gesetze sollten trotzdem in Zotero landen, damit sie auch von den Formatierungen via CSL profitieren. Alternativ händisch anlegen und mit Hyperlink rundum versehen; das lokale Formatting sollte auch durchgeschleift werden, was aber noch zu beweisen ist.

**Comment by sviatoplok** [comment]
Hm, jene Belegzitate, die wir nicht über Zotero machen, müssen wir aber wohl doch formatieren, also wenigstens wie bisher die Kapitälchen setzen, oder? 
Beispiel Ava., Lgbl. etc.

**Comment by skurzinz** [comment]
Ja. Die LGBl. habe ich letzte Woche versucht anzuschauen ob ich das rüberbekomme in Zotero, aber auch die Nachfrage bei der onb hat nichts ergeben...  Als zumindest vorerst: ja.  
Formatieren als *Buchtitel,Quellenbeleg* und bei Bedarf/Möglichkeit einen manuellen Hyperlink auf das betreffende Material rundherum.
